profiles/angular/skeleton/grails-app/assets/javascripts/@grails.codegen.defaultPackage.path@/app.js

```diff
@@ -0,0 +1,14 @@
+//= wrapped
+//= require /@grails.codegen.defaultPackage.path@/core/core
+//= require /@grails.codegen.defaultPackage.path@/home/home
+//= require_self
+//= require_tree services
```

Comment:
Given the main app typically wont have angular controllers, etc directly; Does it make sense to remove the requires for controllers, directives, etc ?